www/src/assets/css/01-utils/tokens/_colors.scss

```diff
@@ -1,4 +1,269 @@
 :root {
+  // Tailwind 4.0 colors
```

Comment:
I know it’s lame to use the Tailwind colors, but they have a really even balance and I was too lazy to find a better ramp that works well in light/dark mode (I’m not really a fan of Radix, and a lot of other systems IMO don’t really provide a nice spread)